WcaOnRails/app/controllers/persons_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PersonsController < ApplicationController
+  def index
+    @regions = { 'Continent' => Continent.all.map { |continent| [continent.name, continent.id] },
+                 'Country' => Country.all.map { |country| [country.name, country.id] } }
+
+    params[:region] ||= "all"
```

Comment:
Personally I don't like changing `params[]`, because you lose track of what was actually posted, and what we made. I'd do

`region = params[:region] || "all"`